scripts/Muon/GUI/Common/calculate_pair_and_group.py

```diff
@@ -55,11 +65,20 @@ def _get_pre_processing_params(context, run):
     except KeyError:
         pass
 
-    try:
-        rebin_args = context.gui_variables["Rebin"]
-        pre_process_params["RebinArgs"] = rebin_args
-    except KeyError:
-        pass
+    if rebin:
+        try:
+            if context.gui_variables['RebinType'] == 'Variable' and context.gui_variables["RebinVariable"]:
+                pre_process_params["RebinArgs"] = context.gui_variables["RebinVariable"]
+        except KeyError:
```

Comment:
can I have an example of when this would fail?

Reply:
I think `1,0.5,15,0.1,10` should fail